index.js

```diff
@@ -1,6 +1,7 @@
 var fs = require('fs')
 var path = require('path')
 var os = require('os')
+var detectLibc = require('detect-libc')
```

Comment:
this module seems to spawn a process, which we can't really do on every run of the program.

Reply:
Yeah. And more than one process, in fact.

@ralphtheninja and I were talking about this and realized we don't really need the complexity of `detect-libc`. At least for `leveldown`, it will be sufficient to check for the existence of a `/etc/alpine-release` file, as Alpine is the only distro (that we know / care about) with musl.

Reply:
Oh that's a great workaround. We could even inline that check and keep this module dependency free then :)

Reply:
What do you mean by "inline" here?

Reply:
inline === create a isAlpine() function here that does it vs a new module